3_parameter_efficient_finetuning/lora_adapters.md

```diff
@@ -59,56 +93,13 @@ merged_model.save_pretrained("path/to/save/merged_model")
 tokenizer.save_pretrained("path/to/save/merged_model")
 ```
 
-## Implementation Guide
-
-Start by installing the required packages:
-```bash
-pip install transformers peft accelerate
-```
-
-The `notebooks/` directory contains practical tutorials for implementing different PEFT methods. Begin with `lora_finetuning.ipynb` for a basic introduction, then explore prompt and prefix tuning through their respective notebooks.
-
-When implementing PEFT methods, start with small rank values (4-8) for LoRA and monitor training loss. Use validation sets to prevent overfitting and compare results with full fine-tuning baselines when possible. The effectiveness of different methods can vary by task, so experimentation is key.
-
-## Using TRL with PEFT
+## Next Steps
 
-PEFT methods can be combined with TRL (Transformer Reinforcement Learning) for efficient reinforcement learning fine-tuning. This integration is particularly useful for RLHF (Reinforcement Learning from Human Feedback) as it reduces memory requirements.
-
-You can scale training across multiple GPUs while keeping memory usage efficient. Here's how to set it up:
-
-```python
-from accelerate import Accelerator
-from peft import LoraConfig
-from transformers import AutoModelForCausalLM
-
-# Get current device from accelerator
-accelerator = Accelerator()
-current_device = accelerator.process_index
-
-# Load model with PEFT config
-lora_config = LoraConfig(
-    r=16,
-    lora_alpha=32,
-    lora_dropout=0.05,
-    bias="none",
-    task_type="CAUSAL_LM"
-)
-
-# Load model on specific device
-model = AutoModelForCausalLM.from_pretrained(
-    "your-model-name",
-    load_in_8bit=True,  # Optional: use 8-bit precision
-    device_map={"": current_device},  # Assign to correct device
-    peft_config=lora_config
-)
-```
+⏩ Move on to the [Prompt Tuning](prompt_tuning.md) guide to learn how to fine-tune a model with prompt tuning.
+⏩ Move on the [Load LoRA Adapters Tutorial](./notebooks/load_lora_adapter.ipynb) to learn how to load LoRA adapters.
```

Comment:
Why are there 2 options here? Don't we want to have a single flow through the course?

Reply:
The course does this a lot to allow people to move around at different paces, but I can how it's confusing.